### ISSUE TITLE
only subset of message signed by pgp after upgrade from v0.11

### ISSUE THREAD

**Issue by dcod3d** [anonaddy/docker#187]
#### Steps to reproduce this issue

1. Upgrade from v0.11.1 to v0.13.7
2. Forward email with encryption

#### Expected behaviour

* Receive message encrypted (worked as expected on v0.11.1)

#### Actual behaviour

* Thunderbird error message 
> Only a subset of this message was encrypted using OpenPGP. The readable parts of the message that are already shown were not encrypted. If you click the decrypt button, the contents of the encrypted parts will be shown.
Clicking on the Decrypt button shows the PGP signature of the message
 
* Receive message, but formatting is stripped. All messages start with this:
>Content-Type: multipart/signed; micalg=pgp-sha256;
 protocol="application/pgp-signature"; boundary=XXXXXX
This is an OpenPGP/MIME signed message (RFC 4880 and 3156)

### Configuration

* Docker version (type `docker --version`) : Docker version 20.10.14, build a224086349
* Docker compose version if applicable (type `docker-compose --version`) : docker-compose version 1.29.2
* Platform (Debian 9, Ubuntu 18.04, ...) : Ubuntu  20.04
* System info (type `uname -a`) : Linux  5.15.0-48-generic #54~20.04.1-Ubuntu SMP Thu Sep 1 16:17:26 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
* Include all necessary configuration files : `docker-compose.yml`, `.env`, ...
used included docker-compose file with ANONADDY_SIGNING_KEY_FINGERPRINT set.
### Docker info

```
Client:
 Context:    default
 Debug Mode: false

Server:
 Containers: 4
  Running: 4
  Paused: 0
  Stopped: 0
 Images: 11
 Server Version: 20.10.14
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 3df54a852345ae127d1fa3092b95168e4a88e2f8
 runc version:
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.15.0-48-generic
 Operating System: Ubuntu Core 18
 OSType: linux
 Architecture: x86_64
 CPUs: 1
 Total Memory: 870.4MiB
 Name: 
 ID: IMZK:D6NV:T7VX:BSEU:R5TR:AQRC:HZSF:LXP7:MFYG:Z6QD:GYLU:YQES
 Docker Root Dir: /var/snap/docker/common/var-lib-docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

```

### Logs

```
anonaddy    | Oct 14 09:09:48 mail postfix/smtpd[914]: connect from sonic313-13.consmr.mail.bf2.yahoo.com[74.6.133.123]
anonaddy    | Oct 14 09:09:49 mail postfix/smtpd[914]: 47E6441149: client=sonic313-13.consmr.mail.bf2.yahoo.com[74.6.133.123]
anonaddy    | Oct 14 09:09:49 mail postfix/cleanup[917]: 47E6441149: message-id=<123@mail.yahoo.com>
anonaddy    | Oct 14 09:09:49 mail postfix/qmgr[905]: 47E6441149: from=<*****@yahoo.com>, size=34163, nrcpt=1 (queue active)
anonaddy    | Oct 14 09:09:49 mail postfix/smtpd[914]: disconnect from sonic313-13.consmr.mail.bf2.yahoo.com[74.6.133.123] ehlo=1 mail=1 rcpt=1 data=1 quit=1 commands=5
anonaddy    | Oct 14 09:09:49 mail postfix/smtpd[914]: connect from localhost[127.0.0.1]
anonaddy    | Oct 14 09:09:49 mail postfix/smtpd[914]: E323941160: client=localhost[127.0.0.1]
anonaddy    | Oct 14 09:09:49 mail postfix/cleanup[917]: E323941160: message-id=<123@mail.yahoo.com>
anonaddy    | Oct 14 09:09:49 mail postfix/qmgr[905]: E323941160: from=<XXXXXX>, size=9575, nrcpt=1 (queue active)
anonaddy    | Oct 14 09:09:49 mail postfix/smtpd[914]: disconnect from localhost[127.0.0.1] ehlo=1 mail=1 rcpt=1 data=1 quit=1 commands=5
anonaddy    | Oct 14 09:09:49 mail postfix/pipe[918]: 47E6441149: to=<XXXXXXX>, relay=anonaddy, delay=1.8, delays=1.2/0.01/0/0.62, dsn=2.0.0, status=sent (delivered via anonaddy service)
anonaddy    | Oct 14 09:09:49 mail postfix/qmgr[905]: 47E6441149: removed
anonaddy    | Oct 14 09:09:50 mail postfix/smtp[953]: Trusted TLS connection established to relayhost:587: TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits) key-exchange X25519 server-signature RSA-PSS (2048 bits) server-digest SHA256
anonaddy    | Oct 14 09:09:50 mail postfix/smtp[953]: E323941160: to=<XXXXXXXX>, relay=relayhost:587, delay=0.87, delays=0.01/0.04/0.55/0.27, dsn=2.0.0, status=sent (250 Great success)
anonaddy    | Oct 14 09:09:50 mail postfix/qmgr[905]: E323941160: removed
```

**Comment by willbrowningme** [comment]
After v0.11.2 Laravel began using Symfony mailer instead of Swiftmailer so I had to rewrite all of the encryption and signing logic.

So the issue is only present with Proton Mail bridge on the Docker image and not at all on the hosted version?

**Comment by dcod3d** [comment]
Mostly yes. It also doesn't work on the iOS proton mail app which I don't think is related to the bridge.

It does work when using anonaddy.com and I will try to see if I can deploy the non docker version this weekend on one of my machines.

**Comment by zcyph** [comment]
Today I spun up a fresh instance (not Docker) with the guide at https://anonaddy.com/self-hosting

Unfortunately I'm still seeing the same behavior there. Messages forwarded and PGP encrypted by AnonAddy are not able to be decrypted on Proton mobile app, and get flagged with "Only a subset of this message was encrypted" in Thunderbird+Proton bridge. The Proton webapp works fine.

So this issue does not seem to be specific to the Docker.

**Comment by dcod3d** [comment]
@zcyph Can you open an issue on the main repo? I haven't had time to run a non-docker instance yet.

**Comment by zcyph** [comment]
@dcod3d sure, [here](https://github.com/anonaddy/anonaddy/issues/400)

**Comment by willbrowningme** [comment]
I've just fixed this issue in the latest release - [v0.13.13](https://github.com/anonaddy/anonaddy/releases/tag/v0.13.13).

I solved it by encrypting and signing the message body in one go as opposed to signing first and then encrypting.
